orderedmap.go

```diff
@@ -3,23 +3,10 @@ package orderedmap
 import (
 	"bytes"
 	"encoding/json"
-	"errors"
 	"sort"
 	"strings"
 )
 
-var NoValueError = errors.New("No value for this key")
```

Comment:
This tiny removal broke our build today ;) Please be aware that other people also use this library and when you introduce breaking change, then change the version significantly.  Good luck

Reply:
Yeah sorry for this. I had no idea this library was as popular as it is otherwise I'd have been more careful with the review!

You're right that this change was significant enough to warrant a larger version increase, it seemed because there was no change in the tests for the public interface it was a minor change, but now you point out the removal of the error I can see it's a bigger change than that. Appreciated.

I'm going to add CI to help keep things running smoothly in the future and detect issues as soon as they arise, like #21

Reply:
Thank you @iancoleman . Appreciate!